Description: Set GAP_SO
 _get_shared_lib_filename does not do the job.
Author: Tobias Hansen <thansen@debian.org>

--- a/sage/src/sage/env.py
+++ b/sage/src/sage/env.py
@@ -274,7 +274,7 @@
 var('SINGULAR_SO', SINGULAR_SO)
 
 # locate libgap shared object
-GAP_SO= _get_shared_lib_filename('gap','')
+GAP_SO= "/usr/lib/%s/libgap.so" % sysconfig.get_config_var('MULTIARCH')
 var('GAP_SO', GAP_SO)
 
 # post process
